build.gradle

```diff
@@ -262,7 +272,7 @@ opensearchplugin {
     name 'opensearch-jvector'
     // zip file name and plugin name in ${opensearch.plugin.name} read by OpenSearch when plugin loading
     description 'OpenSearch jVector plugin'
-    classname 'org.opensearch.knn.plugin.JVectorKNNPlugin'
+    classname 'org.opensearch.plugin.JVectorKNNPlugin'
```

Comment:
I think this will break the build

Reply:
I just realize it is a classname not implementation of a dep.
Let me check.

Thanks.

Reply:
I will restore this for now.